theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/Connector.tsx

```diff
@@ -64,30 +72,40 @@ const Container = styled.div<{isSelected: boolean}>`
   }
 
   &:hover {
-    background: ${(props) =>
-      props.isSelected
-        ? connectorTheme.selectedHoverColor
-        : connectorTheme.hoverColor};
+    background: ${({isSelected, isPopoverOpen}) =>
+      isSelected
+        ? CONNECTOR_THEME.selectedColor(isPopoverOpen)
+        : CONNECTOR_THEME.hoverColor(isPopoverOpen)};
   }
 `
+
+const EasingPopover = styled(BasicPopover)`
+  --popover-outer-stroke: transparent;
+  --popover-inner-stroke: ${COLOR_POPOVER_BACK};
+`
+
 type IProps = Parameters<typeof KeyframeEditor>[0]
 
 const Connector: React.FC<IProps> = (props) => {
   const {index, trackData} = props
   const cur = trackData.keyframes[index]
   const next = trackData.keyframes[index + 1]
 
-  const connectorLengthInUnitSpace = next.position - cur.position
-
   const [nodeRef, node] = useRefAndState<HTMLDivElement | null>(null)
 
+  const {isPointerBeingCaptured} = usePointerCapturing(
+    'KeyframeEditor Connector',
+  )
+
   const [popoverNode, openPopover, closePopover, isPopoverOpen] = usePopover(
```

Comment:
Bug here:

https://user-images.githubusercontent.com/593118/166145346-2a143cf9-e34e-4e62-a98e-7b0d750f2725.mp4

Reply:
Fixed. Constrained the curve editor popover to within the horizontal space of the right part of the dope sheet. Added `minX` and `maxX` options to `usePopover` then set these values based on `layoutP.rightDims`.

Reply:
See my latest comment on the PR for a video.